src/otx/core/model/detection.py

```diff
@@ -35,14 +35,18 @@
     from model_api.models.utils import DetectionResult
     from torch import nn
 
-    from otx.algo.detection.base_models import SingleStageDetector
+    from otx.algo.detection.detectors import SingleStageDetector
 
 
 class OTXDetectionModel(OTXModel[DetBatchDataEntity, DetBatchPredEntity]):
     """Base class for the detection models used in OTX."""
 
     input_size: tuple[int, int]
 
+    def __init__(self, model_name: str, *args, **kwargs) -> None:
```

Comment:
Is it right that every `OTXDetectionModel` should be given a `model_name`? I was under the impression that only models with multiple backbones or versions provide this, is that not the case? Also, I think the default should exist even if it is `None`.

Reply:
Yes, for those cases. But I don't think that some models have this and others don't have this is appropriate.
And I have no idea about setting the default (assume it is `atss_mobilenetv2`) because this default is not applied across all detection models.
More, I referred segmentation part below
https://github.com/openvinotoolkit/training_extensions/blob/db24a410e1a605db2915e41581238bf64a03d496/src/otx/core/model/segmentation.py#L41-L52

Reply:
Okay, so we already have segmentation. Actually, this needs a little more discussion. I'm not sure if `model_name` is necessary for a common class. We'll talk more about it later.

Reply:
I agree with that.